### PR TITLE
Use Lando for the database.

### PR DESCRIPTION
**Pull request by tpendragon**
Provides a more consistent database setup between developers.

### CODE REVIEW

File: .lando.yml [hunk 1]
```diff
@@ -12,6 +12,9 @@ services:
     core: orangelight-core-dev
     config:
       dir: "solr/conf"
+  orangelight_database:
+    type: postgres:10
+    portforward: true
```
Comment (christinach) [inline .lando.yml:17]: can we use postgres 12?

**Review by christinach — CHANGES_REQUESTED** [review]

File: .lando.yml (hunk 1 above)
Comment (tpendragon) [reply .lando.yml:17]: Is that what's in prod?

Comment (christinach) [reply .lando.yml:17]: yes

Comment (christinach) [reply .lando.yml:17]: Wait. no in production is 10. I was looking the qa it has 12.

Comment (tpendragon) [reply .lando.yml:17]: Yeah, `PostgreSQL 10.14 (Ubuntu 10.14-1.pgdg16.04+1) on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 5.4.0-6ubuntu1~16.04.12) 5.4.0 20160609, 64-bit`

**Review by christinach — CHANGES_REQUESTED** [review]
@tpendragon when I use this PR and create the database both in development and in test I'm not able to see them in postgres. Is there another step with lando that I'm missing?

File: README.md [hunk 2]
```diff
@@ -44,7 +30,7 @@ rake db:seed
 Production credentials: In production you'll need to add production
```
Comment (christinach) [inline README.md:30]: Before this line 'Production credentials can you add `Connect to the pg console:` `rails dbconsole -p`. Basically what you mentioned in slack.

**Review by christinach — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tpendragon) [reply README.md]: Done. I also removed this line, it didn't really make sense to me.

**Review by christinach — APPROVED** [review]

**Pull request merged**